### PR TITLE
Copy evmjit.dll to output on windows

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -409,10 +413,8 @@ if (JSCONSOLE)
 	add_subdirectory(ethconsole)
 endif ()
 
-if (NOT WIN32)
```
Comment (chfast) [inline CMakeLists.txt]: That fails Windows build.

File: eth/CMakeLists.txt [hunk 2]
```diff
@@ -42,7 +42,7 @@ if (JSCONSOLE)
 endif()
 
 if (DEFINED WIN32 AND NOT DEFINED CMAKE_COMPILER_IS_MINGW)
-	eth_copy_dlls("${EXECUTABLE}" MHD_DLLS)
+	eth_copy_dlls("${EXECUTABLE}" MHD_DLLS EVMJIT_DLLS)
```
Comment (chfast) [inline eth/CMakeLists.txt]: It does not work!!

Comment (subtly) [inline eth/CMakeLists.txt]: MSVC isn't CMAKE_COMPILER_IS_MINGW :)

**Comment by subtly** [comment]
Does eth need to be directly linked to evmjit? Or is evmjit header only?  If it's the former... eth doesn't have any linkage to evmjit. What all needs to link evmjit?

**Comment by arkpar** [comment]
evmjit builds as dll on widows (unlike other libraries which build as static). Every executable that links it directly or indirectly needs the dll to run. This build action copies the dll to the exe output dir

File: eth/CMakeLists.txt (hunk 2 above)
Comment (arkpar) [inline eth/CMakeLists.txt]: @subtly, NOT DEFINED CMAKE_COMPILER_IS_MINGW there, so msvc fits.

Comment (chfast) [inline eth/CMakeLists.txt]: In other parts "compiler match msvc" is used. And it's just simpler.

Comment (arkpar) [inline eth/CMakeLists.txt]: Someone has put it there for a reason and I don't have a mingw setup to test if it works. So I'd prefer not to touch it.

**Comment by gavofyork** [comment]
base? preferably on whizz branch.

**Comment by debris** [comment]
looksgood

**Pull request merged**